content/en/gitops.md

```diff
@@ -1,31 +1,25 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
-tags: ["", "", ""]
+tags: ["methodology"]
 ---
 
 ## What it is
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
-leverages a source control system like Git as the “single source of truth” and 
-extends this practice to applications, infrastructure, and operational procedures.
+GitOps is a set of best practices based on [shared principles](https://github.com/open-gitops/documents/blob/release-v1.0.0/PRINCIPLES.md), applied to a workflow that depends on software agents to continuously and automatically reconcile an actual running system with its desired state declared in an immutably versioned state store like Git.
+These software agents and practices are used to execute a cohesive workflow and extends this practice to applications, infrastructure, and operational procedures.
 
 ## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges 
-such as configuration drift, failed deployments, relying on a system's previous state for success, 
-missing documentation, or unknown development history.
+Existing processes for infrastructure configuration management – including scripting and tightly coupled CI/CD – are often error prone and can face challenges such as configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
```

Comment:
```suggestion
Existing processes for infrastructure configuration management — including scripting and tightly coupled [CI](https://glossary.cncf.io/continuous-integration/)/[CD](https://glossary.cncf.io/continuous-delivery/) — are often error-prone and face many challenges. 
They often lead to configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
```

Broke sentence down for better readability. We have an entry for continuous deployment and delivery. Is linking to one better than to another?

Reply:
Please not introduce 'configuration drift ' as a new term. 

```suggestion
Existing processes for infrastructure configuration management — including scripting and tightly coupled [CI](https://glossary.cncf.io/continuous-integration/)/[CD](https://glossary.cncf.io/continuous-delivery/) — are often error-prone and face many challenges. 

They often lead to failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
```